src/views/AddItem/AddItem.jsx

```diff
@@ -19,7 +32,48 @@ export default function AddItem({ token }) {
 
   const handleSubmit = (e) => {
     e.preventDefault();
-    addItem();
+    const checkForErrors = validate();
+    if (checkForErrors) {
+      setError('Item already in list!');
+      setListItem({
+        name: '',
+        frequency: '7',
+      });
+      setTimeout(() => {
+        setError('');
+      }, 3000);
+      return;
+    } else {
+      addItem();
+      setListItem({
+        name: '',
+        frequency: '7',
+      });
+      setError('Item added successfully!');
+      setTimeout(() => {
+        setError('');
+      }, 3000);
+    }
+  };
+
+  const validate = () => {
```

Comment:
This function works really well and from your PR documentation, you really have really thought about the intended functionality and user experience when going through this interaction!